File: BaekJoon_C/2581/2581/2581.c
#define _CRT_SECURE_NO_WARNINGS
#include <stdio.h>

int main() {
	int M, N;
	scanf("%d", &M);
	scanf("%d", &N);

	int sum = 0;
	int count = 0;
	int min = 0;

	for (int i = M; i <= N; i++) {
		int chk = 0;

		if (i == 1) chk = 1;

		for (int k = 2; k < i; k++) {
			if (i % k == 0) chk = 1;
		}

		if (chk == 0) {
			sum += i;
			count++;
			if (count == 1) {
				min = i;
			}
		}

	}

	if (sum == 0) {
		printf("-1");
		return 0;
	}

	printf("%d\n", sum);
	printf("%d", min);

	return 0;
}

File: BaekJoon_python/2581.py
M = int(input())
N = int(input())

List = []

for i in range(M, N+1) : 
    error = 0
    if i == 1 : pass
    else : 
        for k in range(2, i) : 
            if (i % k == 0) : error += 1
        if (error == 0) : List.append(i)
        
if (len(List) == 0) : print(-1)
else : 
    print(sum(List))
    print(min(List))